mpas_analysis/shared/climatology/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .climatology import get_lat_lon_comarison_descriptor, get_remapper, \
+    get_mpas_climatology_file_names, get_observation_climatology_file_names, \
+    compute_monthly_climatology, compute_climatology, cache_climatologies, \
+    update_start_end_year, add_years_months_days_in_month, \
+    remap_and_write_climatology
```

Comment:
as long as nothing breaks, I am OK with being inconsistent for now. I was worried things with the old syntax would break, but I ran the `streamfunctionMOC` together with the `regridded` tasks in my last test and all went well.